incubator/drone/values.yaml

```diff
@@ -0,0 +1,41 @@
+## Drone
+## ref: https://hub.docker.com/r/drone/drone
+##
+image:
+  registry: docker.io
+  org: drone
+  name: drone
+  tag: 0.5
+  pullPolicy: IfNotPresent
+
+service:
+  http:
+    externalPort: 80
+    internalPort: 8000
+  type: ClusterIP
+
+server:
+  # Drone server configuration. Values in here get injected as environment variables.
+  # See http://readme.drone.io/admin/installation-reference#server-options for a list of possible values.
+  env:
+    DRONE_DEBUG: "false"
```

Comment:
I've found that I must at least add the following to get the default sqlite db driver to work:
```yaml
DRONE_DATABASE_DATASOURCE: drone.sqlite
```
Otherwise drone-server fails to find/ping the database for some reason and crashloops.  It could probably work with the default datasource (`/var/lib/drone/drone.sqlite`) if this chart created a volume, empty or persistent, mounted to `/var/lib/drone`.